web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -203,6 +216,39 @@ const WebCalendar = (props: WebCalendarProps) => {
     return filteredEvents.length > 0;
   }
 
+  const processEvents = (events) => {
+    // Add a color to the event cycling through the list of available colors
+    if (props.eventNames && props.eventNames.length > 0) {
+      props.eventNames.forEach((name, i) =>
+        events.filter(event => event.title === name).map(event =>
+          Object.assign(event, {color: colors[(i + colors.length) % colors.length]})
+        )
+      );
+    }
+    events.forEach(event =>
+      Object.assign(event, {
+        start: event.start.split(" ")[0],
+        startTimezone: event.start.split(" ")[1],
+        end: event.end.split(" ")[0],
+        endTimezone: event.end.split(" ")[1]
+      })
+    );
+    return events;
+  }
+
+  const setEventContent = (eventContent) => {
+    if (eventContent.view.type === "dayGridMonth") {
+      return <>{eventContent.timeText} {eventContent.event.extendedProps.startTimezone} {eventContent.event.title}</>;
+    }
+    else {
+      // Add the timezone to the timeText string e.g. '9:00am - 12:00pm' becomes '9:00am CEST - 12:00pm CEST'
+      return <>
+        {eventContent.timeText.replace(/m/, "m " + eventContent.event.extendedProps.startTimezone)
```

Comment:
I'm not sure if this is due to how the calendar library handles data or something else, but this looks a bit off. Would it be possible to get the correct value from the data we know instead of doing a regex replace?